perma_web/perma/templates/archive/download_to_view_pdf.html

```diff
@@ -1,7 +1,14 @@
-<div class="record-message">
-  <p class="record-message-primary">Perma.cc can't display this file type on mobile, but you can view or download the archived file by clicking below.</p>
-  <p class="record-message-secondary">File type {{mime_type}}</p>
+<div id="playback-interstitial" class="record-message">
+  <p class="hidden record-message-primary">Perma.cc can't display this file type on mobile, but you can view or download the archived file by clicking below.</p>
```

Comment:
To match the one in `templates/archive/iframe.html`:
```suggestion
  <p class="hidden record-message-primary">Perma.cc can’t display this file type on mobile, but you can view or download the archived file by clicking below.</p>
```